### PR TITLE
cleaned up README

### PR DESCRIPTION
**Pull request by parkand1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,23 +42,26 @@ Add the `cdk-eks-blueprint` library as as a dependency to your CDK project.
 }
 ```
 
+Run the following command to install the dependency to your local npm package manager - `npm i @shapirov/cdk-eks-blueprint`
```
Comment (kcoleman731) [inline README.md:45]: Let's match the style of the other commands and use three tics to put the command on its own line. Easier for devs to copy and paste that way ;)

File: README.md [hunk 2]
```diff
@@ -67,13 +70,18 @@ const addOns: Array<ClusterAddOn> = [
...
+new CdkEksBlueprintStack(app, {id: 'east-test-1', addOns: addOns, teams: []}, {
   env: {
-      account: <AWS_ACCOUNT_ID>,
-      region: <AWS_REGION>,
+      account: 'XXXXXXXXXXXX',
+      region: 'us-east-2'
   },
 });
 ```
+Run the following command to confirm there are no issues with our code - `npm run build`. If there are no errors you should see the following
```
Comment (kcoleman731) [inline README.md:80]: same comment here.

**Review by kcoleman731 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (parkand1) [reply README.md]: Fixed in my most recent push

File: README.md (hunk 2 above)
Comment (parkand1) [reply README.md]: Fixed in my most recent push

**Pull request merged**